### ISSUE TITLE
Error after update 2022.6

### ISSUE THREAD

**Issue by pepe59** [nielsfaber/scheduler-component#201]
The error is for both scheduler and alarm integration

```
Detected integration that uses deprecated `async_get_registry` to access device registry, use async_get instead. Please report issue to the custom component author for scheduler using this method at custom_components/scheduler/__init__.py, line 47: device_registry = await dr.async_get_registry(hass)
Detected integration that uses deprecated `async_get_registry` to access device registry, use async_get instead. Please report issue to the custom component author for alarmo using this method at custom_components/alarmo/__init__.py, line 58: device_registry = await dr.async_get_registry(hass)
```

**Comment by laageide** [comment]
I also experience the same issiue with the scheduler component and I have 3.2.9 installed.
Anyways thanks for this great integration :)

**Comment by nielsfaber** [comment]
I saw that not all the occurrences of the `async_get_registry` were fixed in 3.2.9.
So I made a 3.2.10 release which should tackle them all.

**Comment by pepe59** [comment]
I confirm that update 3.2.10 resolved the Scheduler issue. The only problem is the Alarmo component.
A problem has already been created on the Alarmo git. Should I join there too?
If it is a Scheduler, we can close the problem.
Thank you very much

```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:103
First occurred: 09:11:47 (1 occurrences)
Last logged: 09:11:47

Detected integration that uses deprecated `async_get_registry` to access device registry, use async_get instead. Please report issue to the custom component author for alarmo using this method at custom_components/alarmo/__init__.py, line 58: device_registry = await dr.async_get_registry(hass)
```
